doc/ref_arch/openstack/chapters/chapter04.rst

```diff
@@ -1399,6 +1399,21 @@ the figure below.
 
    Containerised OpenStack Services Topology
 
+Containerisation Support
+^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
@pgoyal01 This paragraph is under the section "Containerised OpenStack Services". As it deals with workloads and not OpenStack services, the first title "Containerised OpenStack Services" can be changed by "Containerisation" or something more general  and 2 sub-sections can be created one for "Containerised OpenStack Services and the other one for "Containerised workloads" .
I agree with the content

Reply:
@karinesevilla Addressed changes by creating a high-level section and two sub-sections